misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/OptionalAssertionsSpec.kt

```diff
@@ -0,0 +1,45 @@
+package ch.tutteli.atrium.specs.integration
+
+import ch.tutteli.atrium.api.fluent.en_GB.messageContains
+import ch.tutteli.atrium.api.fluent.en_GB.toThrow
+import ch.tutteli.atrium.api.verbs.internal.expect
+import ch.tutteli.atrium.specs.Fun0
+import ch.tutteli.atrium.specs.describeFunTemplate
+import ch.tutteli.atrium.specs.lambda
+import ch.tutteli.atrium.specs.name
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.Suite
+import ch.tutteli.atrium.translations.DescriptionBasic.TO_BE
+import ch.tutteli.atrium.translations.DescriptionOptionalAssertion.EMPTY
+import java.lang.AssertionError
+import java.util.Optional
+
+abstract class OptionalAssertionsSpec (
+    isEmpty: Fun0<Optional<Any>>,
+    describePrefix: String = "[Atrium] "
+) : Spek({
+
```

Comment:
please include a SubjectLessSpec (can be copied from other specs)

Reply:
Added